irohad/ametsuchi/impl/redis_flat_block_query.hpp

```diff
@@ -24,6 +24,8 @@
 
 #include "model/converters/json_block_factory.hpp"
 
+#include <boost/optional.hpp>
```

Comment:
formatting again

Reply:
Here is also everything ok.

Reply:
ah, yep
thought we have sorted includes